api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -117,6 +361,28 @@ message RateLimitPerRoute {
     IGNORE = 2;
   }
 
+  // The override option determines how the filter handles the cases where there is an override at multiple levels (virtual host, route, cluster weight).
```

Comment:
Please change this to give an indication of which levels are more specific.  For example:

"""
The override option determines how the filter handles the cases where there is an override config at a more specific level than this one (from least to most specific: virtual host, route, cluster weight).
"""

Reply:
Good idea! Done.